### PR TITLE
Add several fixes to the nailgun CLI integration

### PR DESCRIPTION
**Pull request by jvican**

### CODE REVIEW

File: frontend/src/main/scala/bloop/Server.scala [hunk 1]
```diff
@@ -28,13 +28,16 @@ object Server {
...
-    aliasManager.addAlias(new Alias("compile", "Compile projects", classOf[Cli]))
-    aliasManager.addAlias(new Alias("help", "Show help message", classOf[Cli]))
-    aliasManager.addAlias(new Alias("projects", "Shows the loaded projects", classOf[Cli]))
-    aliasManager.addAlias(new Alias("test", "Runs the tests", classOf[Cli]))
-    aliasManager.addAlias(new Alias("exit", "Exits bloop", classOf[Server]))
+    aliasManager.addAlias(new Alias("about", "Show bloop information.", classOf[Cli]))
+    aliasManager.addAlias(new Alias("clean", "Clean project(s) in the build.", classOf[Cli]))
+    aliasManager.addAlias(new Alias("compile", "Compile project(s) in the build.", classOf[Cli]))
+    aliasManager.addAlias(new Alias("test", "Run project(s)' tests in the build.", classOf[Cli]))
+    aliasManager.addAlias(new Alias("Run", "Run a main entrypoint for project(s) in the build.", classOf[Cli]))
```
Comment (Duhemm) [inline frontend/src/main/scala/bloop/Server.scala]: `Run` should probably be lowercase

Comment (jvican) [reply frontend/src/main/scala/bloop/Server.scala]: good catch

**Comment by jvican** [comment]
Feedback has been addressed.

**Comment by jvican** [comment]
😢

**Comment by jvican** [comment]
😄

**Pull request merged**